manifests/params.pp

```diff
@@ -92,13 +92,18 @@
   $email_smtp_user_name      = undef
   $email_smtp_password       = undef
 
+  # Defines whether Foreman or the tasks plugin provides the Dynflow executor
+  $dynflow_in_core = true
```

Comment:
This will mean a major version bump. Not a problem per se, but important we don't forget this.